generic/tests/netperf.py

```diff
@@ -405,7 +407,7 @@ def start_test(server, server_ctl, host, clients, resultsdir, test_duration=60,
         mpstat_index = 0
 
     for protocol in protocols.split():
-        error_context.context("Testing %s protocol" % protocol, logging.info)
+        error_context.context("Testing %s protocol" % protocol, LOG_JOB.info)
```

Comment:
The function `start_test` and `launch_client` are safe for `test.log`, their `test` is `avocado.core.test.Test` type.

Reply:
During my check, I was noticed that `test` is an optional argument of the `start_test` function, therefore I do have some concern about the placement you suggested. Would you like to share your thought to that?

Reply:
@luckyh Actually it can not be `None`, in the function it has several places that must consume `test`(first place is `netperf_base.record_env_version(test, params, host, server_ctl, fd, test_duration)`), that could be my fault cause I remember this probably was merged by myself, which is a wrong implementation. 
Come back to this PR, I would recommend to replace them with `test.log` since anyway without `test` being passed in the function will crash at other lines where `test` is calling.
And the correction for the function itself(reorder the parameters) could be another task.
Having said that, still I am also okay if you prefer to keep `LOG_JOB`.

Reply:
@vivianQizhu ok, I've added a FIXME comment for this function and updated the code.